docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -5858,32 +5768,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 80,
+   "execution_count": 77,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "'/home/mmccarty/sandbox/rapids/cudf/python/cudf/cudf/tests/data/orc/TestOrcFile.test1.orc'"
-      ]
-     },
-     "execution_count": 80,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "import os\n",
     "from pathlib import Path\n",
-    "current_dir = os.path.dirname(os.path.realpath(\"__file__\"))\n",
-    "cudf_root = Path(current_dir).parents[3]\n",
-    "file_path = os.path.join(cudf_root, \"python\", \"cudf\", \"cudf\", \"tests\", \"data\", \"orc\", \"TestOrcFile.test1.orc\")\n",
-    "file_path"
+    "cudf_root = Path(\".\").absolute().parents[3]\n",
+    "orc_file = Path(\"python/cudf/cudf/tests/data/orc/TestOrcFile.test1.orc\")\n",
```

Comment:
On Windows, `Path`'s constructor correctly handles `/` and returns a `WindowsPath` object. The `pathlib` semantics for path handling are different than `os.path` and abstract away a lot of OS-specific pieces like the separator.

```python
>>> Path("some/long/path/sequence")
WindowsPath('some/long/path/sequence')
>>> str(Path("some/long/path/sequence"))
'some\\long\\path\\sequence'
```
Note that the last line returns (escaped) backslashes when converted to a string.

Also, cuDF is not supported on Windows, so this might not matter anyway. WSL behaves like Linux, so that would not be a concern here.